Modules/Segmentation/ConnectedComponents/include/itkRelabelComponentImageFilter.h

```diff
@@ -266,37 +266,7 @@ class ITK_TEMPLATE_EXPORT RelabelComponentImageFilter : public InPlaceImageFilte
 
   struct RelabelComponentObjectType
```

Comment:
This struct now has only one member. Can it be eliminated entirely and replaced by just `ObjectSizeType`?

Reply:
It could. But as it's used in a `std::pair`, the extra verbosity of `second.m_SizeInPixels` vs just `second` is useful.